common/src/main/scala/uk/ac/wellcome/sqs/SQSReader.scala

```diff
@@ -24,75 +24,77 @@ class SQSReader @Inject()(sqsClient: AmazonSQS, sqsConfig: SQSConfig)
   // If the timeout expires before the consumer sends a delete request, the message is unhidden and can be read by another consumer.
 
   def retrieveAndDeleteMessages(
-    process: Message => Future[Unit]): Future[Unit] =
-    Future {
-      blocking {
-        debug(s"Looking for new messages at ${sqsConfig.queueUrl}")
-        receiveMessages()
-      }
-    } flatMap { messages =>
-      if (messages.nonEmpty)
-        info(
-          s"Received messages ${messages.map { _.getMessageId }} from queue ${sqsConfig.queueUrl}")
-      else
-        debug(
-          s"Received messages ${messages.map { _.getMessageId }} from queue ${sqsConfig.queueUrl}")
-      processAndDeleteMessages(messages, process).map { _ =>
+    process: Message => Future[Unit]): Future[Unit] = {
+    val eventuallyProcessMessages =
+      for {
+        _ <- Future.successful {
+          debug(s"Looking for new messages at ${sqsConfig.queueUrl}")
+        }
+        ms <- receiveMessages()
```

Comment:
Not really important but I think I preferred the old variable name `messages`

Reply:
I agree, but I think in this case I used a shorter name because otherwise the proceeding code breaks the 80 character limit and the re-formatter makes it look ugly. Perhaps we could shorten the name of the `processAndDeleteMessage` method instead?

Reply:
What if we made the 80 character limit be a 100 character limit instead?

Reply:
+1

Reply:
pffft yeah, damn the 80 characters if it means we get better variable names.